src/5e-SRD-Magic-Items.json

```
@@ -244,6 +244,90 @@
+    "name": "Bag of Holding",
+    "equipment_category": {
+      "name": "Wondrous Item",
+      "url": "/api/equipment-categories/wondrous-item"
+    },
+    "desc": [
+      "Wondrous	item,	uncommon",
+      "This bag has an interior space considerably larger than its outside dimensions, roughly 2 feet in diameter at the mouth and 4 feet deep. The bag can hold up to 500 pounds, not exceeding a volume of 64 cubic feet. The bag weighs 15 pounds, regardless of its contents. Retrieving an item from the bag requires an action.",
+      "If the bag is overloaded, pierced, or torn, it ruptures and is destroyed, and its contents are scattered in the Astral Plane. If the bag is turned inside out, its contents spill forth, unharmed, but the bag must be put right before it can be used again. Breathing creatures inside the bag can survive up to a number of minutes equal to 10 divided by the number of creatures (minimum 1 minute), after which time they begin to suffocate.",
+      "Placing a bag of holding inside an extradimensional space created by a Handy Haversack, Portable Hole, or similar item instantly destroys both items and opens a gate to the Astral Plane. The gate originates where the one item was placed inside the other. Any creature within 10 feet of the gate is sucked through it to a random location on the Astral Plane. The gate then closes. The gate is one-way only and can't be reopened."
+    ],
+    "url": "/api/magic-items/bag-of-holding"
+ },
+  {
+    "index": "bag-of-tricks",
+    "name": "Bag of Tricks",
+    "equipment_category": {
+     "name": "Wondrous Item",
+      "url": "/api/equipment-categories/wondrous-item"
+    },
+    "desc": [
+      "Wondrous	item,	uncommon",
+      "This ordinary bag, made from gray, rust, or tan cloth, appears empty. Reaching inside the bag, however, reveals the presence of a small, fuzzy object. The bag weighs 1/2 pound.",
+      "You can use an action to pull the fuzzy object from the bag and throw it up to 20 feet. When the object lands, it transforms into a creature you determine by rolling a d8 and consulting the table that corresponds to the bag's color. The creature vanishes at the next dawn or when it is reduced to 0 Hit Points.",
+      "The creature is friendly to you and your companions, and it acts on Your Turn. You can use a Bonus Action to Command how the creature moves and what action it takes on its next turn, or to give it general orders, such as to Attack your enemies. In the absence of such orders, the creature acts in a fashion appropriate to its Nature.",
+      "Once three fuzzy Objects have been pulled from the bag, the bag can't be used again until the next dawn.",
+      "| Gray Bag |",
+      "| d8 | Creature |",
+      "|---|---|",
+      "| 01 | Weasel |",
+      "| 02 | Giant Rat |",
+      "| 03 | Badger |",
+      "| 04 | Boar |",
+      "| 05 | Panther |",
+      "| 06 | Giant Badger |",
+      "| 07 | Dire Wolf |",
+      "| 08 | Giant Elk |",
+      
+      
```

These lines should be:
```
"",
"",
```

---

Oh! I can do suggestions!
```suggestion
      "",
      "",
```